pkgs/development/python-modules/manim-voiceover/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, pytest-xdist
+, pytestCheckHook
+, manim
+, mutagen
+, pydub
+, humanhash3
+, sox
+, pip
+, python-dotenv
+, gtts
+, pynput
+, pyaudio
+, playsound
+, stable-ts
+, pygobject3
+, gst_all_1
+, wrapGAppsHook
+, intltool
+}:
+
+buildPythonPackage rec {
+  pname = "manim-voiceover";
+  format = "pyproject";
+  version = "0.3.0";
+  disabled = pythonOlder "3.7";
+
+  extras = [ "gtts" "recorder" ];
```

Comment:
This should probably be in passthru.optional-dependencies

Reply:
Done